iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/compaction/IcebergCompactionUtil.java

```diff
@@ -55,6 +60,12 @@ public static boolean shouldIncludeForCompaction(Table table, String partitionPa
             table.specs().get(file.specId()).partitionToPath(file.partition()).equals(partitionPath);
   }
 
+  public static boolean shouldIncludeForCompaction(Table table, String partitionPath, ContentFile<?> file,
```

Comment:
are we checking only data files?

Reply:
I missed it. Updated to include will in compaction if it meets file size threshold or if any deletes exist.